### ISSUE TITLE
Hosting Learning Path on new InnerSource Commons website

### ISSUE THREAD

**Issue by tsadler1988** [InnerSourceCommons/InnerSourceLearningPath#362]
Assist with https://github.com/InnerSourceCommons/innersourcecommons.net/issues/28

**Comment by tsadler1988** [comment]
Started working on this:
- Scripts: #369 
- New website: https://github.com/InnerSourceCommons/innersourcecommons.net/pull/32

**Comment by tsadler1988** [comment]
Note https://github.com/InnerSourceCommons/innersourcecommons.net/issues/51. 

I assume we won't want to sign off until translations are hosted too? This work has been started under https://github.com/InnerSourceCommons/innersourcecommons.net/pull/52.

**Comment by tsadler1988** [comment]
Remaining task - when selecting a language, we want there to be some feedback for the user. Either:
- Transalte index.md #382
- Add placeholder index.md files to make language selection visible and obvious

**Comment by tsadler1988** [comment]
> Add placeholder index.md files to make language selection visible and obvious

I will do this as part of this issue to complete MVP Learning Path for the new website.

**Comment by claredillon** [comment]
The placeholders have all been edited by @clcoffey to include text to indicate that translations are coming soon.

**Comment by tsadler1988** [comment]
Looks good to me @claredillon. OK to close this issue from my point of view, but I might wait until Thursday's meeting and double check with others.

**Comment by rrrutledge** [comment]
I see translations there - what do you mean about coming soon?

**Comment by tsadler1988** [comment]
> I see translations there - what do you mean about coming soon?

The index pages i.e. the description of each section e.g. https://github.com/InnerSourceCommons/InnerSourceLearningPath/issues/382 completed the German index pages translations. On reflection this might be misleading - people may think this means nothing is translated. I believe @RJPlog found this confusing too.

We could change the text, or push to get index page translations ASAP.

**Comment by clcoffey** [comment]
Hi @tsadler1988 and @rrrutledge I tried adding a larger piece of text there that said something along the lines of: This is the index page for the <language> translation of the Learning Path. The index page is in the process of being translated. Press the "Learn" Button below to access the articles which have already been translated.

However, when I added this text the original English text disappeared from the index page. The only way I managed to keep both was by putting all the text in the description: section at the top of each page. In this case, there was no way I could find to format it differently and separate it from the untranslated English text. I would love some suggestions on how we could make it better while we are waiting for translations. Tom if you manage to change one of the pages and be happy with it I can help you with changing the rest.
